esx_service/utils/auth.py

```diff
@@ -42,6 +46,67 @@ def get_auth_mgr():
         thread_local._auth_mgr.connect()
     return thread_local._auth_mgr
 
+def get_fallback_tenant():
+    """ 
+        Get __FALLBACK tenant by querying the auth DB.
+        VMs which does not belong to any tenant explicitly will 
+        be assigned to __FALLBACK tenant if __FALLBACK tenant exists
+        Return value:
+        -- error_info: return None on success or error info on failure
+        -- tenant_uuid: return __FALLBACK tenant uuid on success,  
+           return None on failure or __FALLBACK tenant does not exist
+        -- tenant_name: return __FALLBACK tenant name on success,
+           return None on failure or __FALLBACK tenant does not exist
+    """
+    tenant_uuid = None
+    _auth_mgr = get_auth_mgr()
+    try:
+        cur = _auth_mgr.conn.execute(
```

Comment:
I am not going to address it in this PR, but will file an issue to track this enhancement.